runtime/README.md

```diff
@@ -458,19 +458,19 @@ Table: `DPE_TAG_TCI` input arguments
 | **Name**     | **Type**      | **Description**
 | --------     | --------      | ---------------
 | chksum       | u32           | Checksum over other input arguments, computed by the caller. Little endian.
-| handle       | u8[16]        | DPE context handle 
-| tag          | u32           | A unique tag which the handle will be associated with 
+| handle       | u8[16]        | DPE context handle
+| tag          | u32           | A unique tag which the handle will be associated with
```

Comment:
Optional uber-nit: "with which the handle will be associated"

Reply:
Will handle in a follow up when I have more time to spend. For now I'm just looking to get the content all there. Stephanie will help with a round of edits.